docs/hopsworksai/azure/getting_started.md

```diff
@@ -352,6 +348,111 @@ Click on *Save* (7).
   </figure>
 </p>
 
+## Optional Step: Add upgrade permissions to your user assigned managed identity
```

Comment:
I'm not sure what to think about the changes made to this page. I think I get why you are trying to do. But, I would prefer the getting started to be as simple as possible and to have all the optional complexity described in more advanced part of the documentation.

Reply:
I agree in that case, i would just keep this change as part of the upgrade documentation instead